docs/releases/breaking-changes.md

```diff
@@ -13,6 +13,81 @@ these to a minimum, but rest assured we only make them with good reason.
 
 Starting with Directus 10.0, here is a list of potential breaking changes with remedial action you may need to take.
 
+## Version 10.10.0
+
+### Session Cookie Based Authentication
+
+For improved security and ease of use we have implemented session based authentication and have updated the App to use
+this over the previous token based authentication. This impacts `oauth2`, `open-id` and `saml` SSO installations as they
+too will now also default to the new session based authentication in order to work with the App out-of-the-box.
+
+To keep using the old SSO functionality setting the refresh token instead of session token you can set
```

Comment:
We need to mention that this would only work for external apps and not when used for the Data Studio

Reply:
If there should be a need for both methods, I think it might be possible to set-up the same provider two times under a different name 🤔